src/main/java/org/launchcode/water_garden_tour/controllers/MapController.java

```diff
@@ -1,16 +1,20 @@
 package org.launchcode.water_garden_tour.controllers;
 
+import org.launchcode.water_garden_tour.models.data.GardenRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
 public class MapController {
 
+    @Autowired
+    GardenRepository gardenRepository;
+
     @RequestMapping("map")
     public String map(Model model) {
-
-        model.addAttribute("title","Kansas City Water Garden Tours");
```

Comment:
It the title not needed anymore? Can we keep both?

Reply:
The title is getting pulled in from our fragment:head, so there was no need for it here.